oemof_b3/tools/data_processing.py

```diff
@@ -565,6 +567,121 @@ def aggregate_timeseries(df, columns_to_aggregate, agg_method=None):
     return df_aggregated
 
 
+def prepare_attr_name(sc, regions, overwrite):
```

Comment:
This function should only set the name according to region, carrier, tech provided in the scalars. No need to pass regions.

Reply:
You mean `set_name` function, don't you? Unfortunately, I don't know how to implement this so that it is simplified and not more complicated. Could you maybe write a suggestion?